crypto/client.go

```diff
@@ -15,8 +16,16 @@ func New(config Config) (Client, error) {
 		if config.Type != "" {
 			fmt.Println("Invalid encryption type, using default")
 		}
-		return NewAESOpenPGPClient(config)
-	case "aes-openpgp":
-		return NewAESOpenPGPClient(config)
+		fallthrough
+	case "aes_openpgp":
+		// Remove other settings
+		// TODO(leon): This is shitty.
+		config.Settings.AESGCMSettings = nil
```

Comment:
We need to nil out all non-relevant settings, i.e. settings which do not affect the current crypto type.

Example:
When using the AES-GCM type, the safe should not contain any settings for the OpenPGP mode.